src/main/java/randoop/generation/UniformRandomMethodSelection.java

```diff
@@ -0,0 +1,29 @@
+package randoop.generation;
+
+import java.util.List;
+import randoop.operation.TypedOperation;
+import randoop.util.Randomness;
+
+/** This class selects a method with uniform probability. */
+public class UniformRandomMethodSelection implements TypedOperationSelector {
+  private final List<TypedOperation> operations;
+
+  /**
+   * Saves a reference to the list of methods under test.
```

Comment:
The main thing this does is to create a new object.  Please reflect that in the documentation.